os/mm/kmm_heap/kmm_calloc.c

```diff
@@ -67,22 +67,15 @@
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
-#ifdef CONFIG_DEBUG_MM_HEAPINFO
+
 static void *kheap_calloc(size_t n, size_t elem_size, mmaddress_t retaddr)
-#else
-static void *kheap_calloc(size_t n, size_t elem_size)
-#endif
 {
 	int heap_idx;
 	void *ret;
 	struct mm_heap_s *kheap = kmm_get_baseheap();
 
 	for (heap_idx = HEAP_START_IDX; heap_idx <= HEAP_END_IDX; heap_idx++) {
-		ret = mm_calloc(&kheap[heap_idx], n, elem_size
-#ifdef CONFIG_DEBUG_MM_HEAPINFO
-				, retaddr
-#endif
-				);
+		ret = mm_calloc(&kheap[heap_idx], n, elem_size, retaddr);
 		if (ret != NULL) {
 			return ret;
 		}
```

Comment:
Please check the `mm_manage_alloc_fail` as well in each allocation API.
In this file, line 85.